likeminds-feed-reactnative-integration/components/LMMedia/LMCarousel/index.tsx

```diff
@@ -49,6 +50,29 @@ const LMCarousel = React.memo(
       }
     };
 
+    const getMaxHeightOfAttachments = () => {
+      if (!post?.attachments?.length) return 0;
+    
+      const ScreenWidth = Dimensions.get("window").width;
```

Comment:
```suggestion
      const screenWidth = Dimensions.get("window").width;
```
1. Rename variable
2. Why are we calculating screenWidth again and again, calculate once and store it for better performance. Change this approach.

Reply:
this value is internally cached as its only calculated on app launch or on orientation changes

Reply:
But there is so much code duplication, and the code is unoptimized. We should write it properly.